pkg/kubelet/prober/worker.go

```diff
@@ -105,13 +108,14 @@ func (w *worker) run() {
 		w.probeManager.removeWorker(w.pod.UID, w.container.Name, w.probeType)
 	}()
 
+	tickPeriod := time.Duration(w.spec.PeriodSeconds) * time.Second
 probeLoop:
 	for w.doProbe() {
 		// Wait for next probe tick.
 		select {
 		case <-w.stop:
 			break probeLoop
-		case <-probeTicker.C:
+		case <-time.Tick(wait.Jitter(tickPeriod, workerProbeJitterFactor)):
```

Comment:
Right, `time.After` seems to solve the leak (at least documentation does not state it otherwise).